howto/anbox/auth.md

```diff
@@ -0,0 +1,105 @@
+(howto-auth)=
+
+## Configure user permissions in Anbox Cloud
+
+## Configure OpenFGA
+
+```{important}
+This section is required only for the charmed deployment. For the Anbox Cloud Appliance, you can skip this step and proceed to creating identities.
+```
+Anbox Management Service(AMS) requires a store ID in OpenFGA to manage authorization data. To generate a store ID in OpenFGA, follow the steps in the [OpenFGA documentation](https://openfga.dev/docs/getting-started/create-store) and create a store. 
+
+When you have a store ID, run these commands to configure AMS:
+
+```
+amc config set openfga.api.url '<URL of your OpenFGA instance>'
+amc config set openfga.store.id '<store id of the user created store>'
+```
+
+Optionally, you can also configure the API token if your deployment setup will require authentication for API access to OpenFGA:
+
+    amc config set openfga.api.token '\<api token generated from OpenFGA\>'
+
+When the API URL and store ID are set, AMS starts synchronizing the data with OpenFGA. You might see some authorization failures initially till the synchronization completes.
+
+## Create identities
+
+To create an OIDC identity, run:
+
+    $ amc auth identity create oidc/test@example.com
```

Comment:
Do we prefix commands with `$`? It's not the case above, but it is the case here. It doesn't like very consistent right now.